lighthouse-core/scripts/manual-chrome-launcher.js

```diff
@@ -14,29 +13,36 @@
  *     chrome-debug --enable-extensions
  */
 
-const {launch} = require('chrome-launcher');
+const {Launcher, launch} = require('chrome-launcher');
 
 const args = process.argv.slice(2);
-let chromeFlags;
+const chromeFlags = [];
 let startingUrl;
 let port;
-let enableExtensions;
+let shouldEnableExtensions;
```

Comment:
seems like these two variables always have the same value, yes? 

feels like we could ditch shouldEnableExtensions at this point if it's just the `if`